mediorum/ddl/drop_uploads_audio_analysis_ops.sql

```diff
@@ -0,0 +1,8 @@
+begin;
+    delete from ops where "table" = 'uploads' and action = 'update' and exists (
+        select 1
+        from jsonb_array_elements(data) as elem
+        where elem->>'audio_analysis_status' = 'timeout'
+        and elem->'selected_preview' = '{"Valid": false, "String": ""}'
```

Comment:
selected_preview clause is because the only potential update to the record after the audio analysis update is if the user changes the preview.